content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -100,10 +96,17 @@ spec:
 EOF
 {{< /text >}}
 
-Apply the configuration in the `istio-system` namespace of `external_cluster`:
+Install the configuration to create the ingress gateway in the `istio-system` namespace of `external_cluster`:
 
 {{< text bash >}}
-$ istioctl apply -f controlplane-gateway.yaml --context="${CTX_EXTERNAL_CLUSTER}"
+$ istioctl install -f controlplane-gateway.yaml --context="${CTX_EXTERNAL_CLUSTER}"
```

Comment:
Another good catch!  This made me want to add istioctl apply cmd to alias to istioctl install :)